jobs/JREGIONAL_RUN_MET_ENSEMBLESTAT_VX_GRID

```diff
@@ -63,7 +63,7 @@ on gridded fields by initialization time for all forecast hours.
 #
 #-----------------------------------------------------------------------
 #
-$SCRIPTSdir/exregional_run_vx_ensgrid.sh || \
+$SCRIPTSdir/exregional_run_MET_EnsembleStat_vx_grid.sh || \
```

Comment:
@gsketefian NCO requires that ex-scripts be named all lowercase so this and others need to change.

Reply:
@danielabdi-noaa Ok, ex-script names have been changed, will push commit soon.